lib/ProductOpener/Nutrition.pm

```diff
@@ -2040,4 +2063,26 @@ sub filter_out_nutrients_not_in_taxonomy ($product_ref) {
 	return;
 }
 
+=head2 convert_sodium_to_salt ( $sodium_value )
+
+Converts a sodium value to its equivalent salt value using the EU standard conversion factor (2.5).
+
+=cut
+
+sub convert_sodium_to_salt ($sodium_value) {
+
+	return $sodium_value * 2.5;
+}
```

Comment:
Great to have made sub for that, it's really important to "localize" this kind of knowledge.